### PR TITLE
Chapter 3 Markdown with Images (#46)

### PR DESCRIPTION
**Pull request by atc-vince**
Signed-off-by: atc-vince <atchyperledger@gmail.com>

Added Chapter 3 images and fixed Supply Chain II TOC link

Signed-off-by: atc-vince <atchyperledger@gmail.com>

Chapter 3 Markdown and commits squashed

### CODE REVIEW

File: LFS171x/docs/the-promise-of-business-blockchain-technologies.md [hunk 1]
```diff
@@ -1 +1,347 @@
...
+    - [Knowledge Check 3.3](#knowledge-check-33)
+  - [Conclusions and Learning Objectives (Review)](#conclusions-and-learning-objectives-review)
+    - [Learning Objectives (Review)](#learning-objectives-review)
+    - [Conclusions](#conclusions)
+
+<!-- /TOC -->
+
+## Introduction and Learning Objectives
+
+### Introduction (Nathalie Salami)
```
Comment (tkuhrt) [inline LFS171x/docs/the-promise-of-business-blockchain-technologies.md]: Can we add "Video: " to the beginning of this title? I notice that my chapter left this out too. I need to go fix that.

File: LFS171x/docs/the-promise-of-business-blockchain-technologies.md [hunk 2]
```diff
@@ -1 +1,347 @@
...
+
+In permissioned blockchains, a consortium of organizations are responsible for authenticating and controlling the participants in a blockchain. In public blockchains, no central authority or administration is required to exchange data. Blockchains can drive business innovation through controlled data-sharing networks for industry consortiums.
+
+The promise of distributed ledger technologies (DLT) to simplify and automate key work functions has many industries taking notice. Businesses recognize the efficiency gains from transitioning from closed and proprietary solutions to standard open source capabilities, such as Hyperledger business blockchain technologies. Several common project features of blockchain applications are taking shape as the technology matures.
+
+How exactly are businesses using these emerging technologies today? Next, we will explore the state of distributed ledger technologies in actual corporate settings, and how they compare against traditional tools. 
+
+### Hyperledger Advantages for Businesses (Brian Behlendorf)
```
Comment (tkuhrt) [inline LFS171x/docs/the-promise-of-business-blockchain-technologies.md]: Can we add "Video: " to the beginning of this title?

File: LFS171x/docs/the-promise-of-business-blockchain-technologies.md [hunk 3]
```diff
@@ -1 +1,347 @@
...
+
+[![Hyperledger Advantages for Businesses (Brian Behlendorf)](../images/video-image.png)](https://youtu.be/V65RHZ-vtD8)
+
+### Blockchain Use Case (Brian Behlendorf)
```
Comment (tkuhrt) [inline LFS171x/docs/the-promise-of-business-blockchain-technologies.md]: Can we add "Video: " to the beginning of this title?

File: LFS171x/docs/the-promise-of-business-blockchain-technologies.md [hunk 4]
```diff
@@ -1 +1,347 @@
...
+
+A blockchain may record a hash of a document. As an example, photographers could place a hash of their unique digital photographs on the blockchain. The hash of a digital photograph will be constant so long as the photograph file has not been altered. Therefore, the blockchain can control and track the distribution of the photograph, detect the introduction of counterfeit images, and be used to resolve disputes as to who first introduced the image. By placing a hash of intellectual property documents on the blockchain, a party can publicly demonstrate data ownership, and prove the existence of certain documents at a given moment in time, without revealing the actual data. In addition to the hash, you may also choose to store the location of the file in the blockchain, which could be used for retrieval.
+
+### Business Interest in Hyperledger (Brian Behlendorf)
```
Comment (tkuhrt) [inline LFS171x/docs/the-promise-of-business-blockchain-technologies.md]: Can we add "Video: " to the beginning of this title?

File: LFS171x/docs/the-promise-of-business-blockchain-technologies.md [hunk 5]
```diff
@@ -1 +1,347 @@
...
+|![The_BlockRx_Pharma_Ecosystem](../images/the-promise-of-business-blockchain-technologies/The_BlockRx_Pharma_Ecosystem.png)|
+|:--:|
+
+<p align="center">
+  By <a href="https://www.blockrx.com/">The BlockRx Project</a>, retrieved from the <a href="https://wiki.hyperledger.org/groups/healthcare/healthcare-wg">Hyperledger Healthcare Working Group, August 22nd, 2017 Meeting</a>
+</p>
+
+## When to Use or Not to Use Blockchain Technologies
+
+### What Enterprises Look for When Evaluating Whether or Not to Use Hyperledger (Brian Behlendorf)
```
Comment (tkuhrt) [inline LFS171x/docs/the-promise-of-business-blockchain-technologies.md]: Can we add "Video: " to the beginning of this title?

File: LFS171x/docs/the-promise-of-business-blockchain-technologies.md [hunk 6]
```diff
@@ -1 +1,347 @@
...
+
+The trade finance industry can also leverage information visible in a supply chain blockchain. In its broadest sense, trade finance manages capital required for international trade. Trade financing has become the norm for cross border transactions, with the [World Trade Organization](https://www.wto.org/english/res_e/booksp_e/tradefinsme_e.pdf) estimating that "up to 80 percent of global trade is supported by some sort of financing or credit insurance" (2016). An exporter needs to to mitigate the risk of non-payment, while an importer wants to mitigate the supply risk. The function of trade finance is to act as a third party to remove the payment risk and the supply risk, whilst providing the exporter with accelerated receivables, and the importer with extended credit. Institutions that provide capital during these trades can leverage the information visible in a supply chain blockchain to better evaluate companies for lending.
+
+|![Supply Chain of Tuna](../images/the-promise-of-business-blockchain-technologies/Supply_Chain_of_Tuna.png)|
+|:--:|
+
+<p align="center">
```
Comment (tkuhrt) [inline LFS171x/docs/the-promise-of-business-blockchain-technologies.md]: To match other chapters, use the following format
```
|![Supply Chain of Tuna](../images/the-promise-of-business-blockchain-technologies/Supply_Chain_of_Tuna.png)|
|:--:|
| *Source: Nishan Degnarain (used with permission)* |
```

File: LFS171x/docs/the-promise-of-business-blockchain-technologies.md [hunk 7]
```diff
@@ -1 +1,347 @@
...
+
+Provenance is a record of ownership used as a guide to authenticity or quality. Because of the overhead involved in traditional provenance records, they were only available for very large ticket items, such as works of art. With the efficiencies gained from blockchain technology, provenance records can be available for a wider range of goods. This improved information can aid consumers as they make purchasing decisions.
+
+|![Provenance Use Case](../images/the-promise-of-business-blockchain-technologies/Provenance_Use_Case.jpg)|
+|:--:|
+
+<p align="center">
+How Provenance Works (by [Project Provenance Ltd](https://www.provenance.org/), used with permission)
+</p>
+
```
Comment (tkuhrt) [inline LFS171x/docs/the-promise-of-business-blockchain-technologies.md]: To match other chapters, use:
```
|![Provenance Use Case](../images/the-promise-of-business-blockchain-technologies/Provenance_Use_Case.jpg)|
|:--:|
| *How Provenance Works (by [Project Provenance Ltd](https://www.provenance.org/), used with permission)* |
```

File: LFS171x/docs/the-promise-of-business-blockchain-technologies.md [hunk 8]
```diff
@@ -1 +1,347 @@
...
+As the previous section on blockchains for supply chain management illustrated, blockchain data improves insight into products, as they move through their lifecycle. Large enterprises are not the only parties to benefit from this increased visibility. Consumers can also benefit from blockchain technology.
+
+Provenance is a record of ownership used as a guide to authenticity or quality. Because of the overhead involved in traditional provenance records, they were only available for very large ticket items, such as works of art. With the efficiencies gained from blockchain technology, provenance records can be available for a wider range of goods. This improved information can aid consumers as they make purchasing decisions.
+
+|![Provenance Use Case](../images/the-promise-of-business-blockchain-technologies/Provenance_Use_Case.jpg)|
+|:--:|
+
+<p align="center">
+How Provenance Works (by [Project Provenance Ltd](https://www.provenance.org/), used with permission)
```
Comment (tkuhrt) [inline LFS171x/docs/the-promise-of-business-blockchain-technologies.md]: For some strange reason, this is not showing up as a link for me when I view it. Instead, it shows up as:
```
How Provenance Works (by [Project Provenance Ltd](https://www.provenance.org/), used with permission)
```

I don't see anything wrong with it.

File: LFS171x/docs/the-promise-of-business-blockchain-technologies.md [hunk 9]
```diff
@@ -1 +1,347 @@
...
+
+### Property Rights (Part II)
+
+Companies may use blockchain technologies to record ownership rights and responsibilities. Specifically, governments have put land registry records on blockchain (Laura Shin, [forbes.com](https://www.forbes.com/sites/laurashin/2016/04/21/republic-of-georgia-to-pilot-land-titling-on-blockchain-with-economist-hernando-de-soto-bitfury/#1c7a07944da3), 2016). Companies have also put intellectual property registration and ownership on blockchain (poex.io). Intellectual property includes copyright, trademark, and patents. To legally protect ownership rights in these, one registers their production, or invention, or otherwise proves when the work was established, and that they are the origin of the work.
```
Comment (tkuhrt) [inline LFS171x/docs/the-promise-of-business-blockchain-technologies.md]: Missing link for poex.io. Should be 
```
([poex.io](https://poex.io/about))
```

File: LFS171x/docs/the-promise-of-business-blockchain-technologies.md [hunk 10]
```diff
@@ -1 +1,347 @@
...
+
+|![Property Rights](../images/the-promise-of-business-blockchain-technologies/PROPERTY_RIGHTS.jpg)|
```
Comment (tkuhrt) [inline LFS171x/docs/the-promise-of-business-blockchain-technologies.md]: Missing title on this picture. EdX has this as the title, which is not part of the image:
```
REGISTERING PROPERTY TITLES ON A BLOCKCHAIN VIA A SMART CONTRACT 
```

File: LFS171x/docs/the-promise-of-business-blockchain-technologies.md [hunk 11]
```diff
@@ -1 +1,347 @@
...
+
+A blockchain may record a hash of a document. As an example, photographers could place a hash of their unique digital photographs on the blockchain. The hash of a digital photograph will be constant so long as the photograph file has not been altered. Therefore, the blockchain can control and track the distribution of the photograph, detect the introduction of counterfeit images, and be used to resolve disputes as to who first introduced the image. By placing a hash of intellectual property documents on the blockchain, a party can publicly demonstrate data ownership, and prove the existence of certain documents at a given moment in time, without revealing the actual data. In addition to the hash, you may also choose to store the location of the file in the blockchain, which could be used for retrieval.
+
+### Business Interest in Hyperledger (Brian Behlendorf)
+
+[![Business Interest in Hyperledger (Brian Behlendorf)](../images/video-image.png)](https://youtu.be/cDZg8R5z1oQ)
+
+### Finance (Part I)
+
+<p align="center">
```
Comment (tkuhrt) [inline LFS171x/docs/the-promise-of-business-blockchain-technologies.md]: For Quotes, use the following syntax:
```
 > "Blockchain has huge potential to move the financial services industry away from messaged based models, slow reconciliation processes and inefficiency of fragmented data stores. With blockchain, financial services can move to a shared data construct, driving down costs, increasing efficiency and opening up entirely new business models."

-David Treat, Accenture
```

File: LFS171x/docs/the-promise-of-business-blockchain-technologies.md [hunk 12]
```diff
@@ -1 +1,347 @@
...
... the first blockchain use case in existence was payments. However, Bitcoin proved to be quite slow to process payments, "somewhere in the region of 7 transactions per second" ([Guy Brandon, due.com](https://due.com/blog/can-the-blockchain-scale/), February 2017), when compared to Visa, which "averages around 2,000 transactions per second, with peak capacity of perhaps 50,000 transactions per second" ([Guy Brandon, due.com](https://due.com/blog/can-the-blockchain-scale/), February 2017). Developers are actively working to increase the throughput capacity of Bitcoin and other blockchain payment systems ([lightning.network](https://lightning.network/)). Payments, especially international payments, can be quite costly. Blockchain technologies plan to decrease the costs associated with payments, by allowing parties to interact directly, instead of transferring through an intermediary, such as a bank. In addition, having a record of all past payments is useful to auditors and regulators. Financial institutions have heavily researched blockchain payment systems because a universally recorded world state of payment information can decrease the number of payment disputes among institutions.
```
Comment (tkuhrt) [inline LFS171x/docs/the-promise-of-business-blockchain-technologies.md]: add italics `_italics_` to "peer-to-peer electronic cash system", "somewhere in the region of 7 transactions per second", and "averages around 2,000 transactions per second, with peak capacity of perhaps 50,000 transactions per second"

As in:
```
"_peer-to-peer electronic cash system_"
"_somewhere in the region of 7 transactions per second_"
"_averages around 2,000 transactions per second, with peak capacity of perhaps 50,000 transactions per second_"
```

File: LFS171x/docs/the-promise-of-business-blockchain-technologies.md [hunk 13]
```diff
@@ -1 +1,347 @@
...
+
+### Healthcare (Part I)
+
+A number of multi-party processes in the healthcare industry can leverage distributed ledger technology. By streamlining these multi-party processes, the healthcare industry can reduce the time and expense of collecting and verifying multiple pieces of information in order to deliver quality care to patients. Healthcare providers and insurance companies have begun to explore how blockchain can improve the delivery of patient care.
+
+In 2015, the US spent 27.42% of the federal budget, or $1.05 trillion, on healthcare ([National Priorities Project](https://www.nationalpriorities.org/budget-basics/federal-budget-101/spending/)). Because these costs are so high, the US government, in particular, has invested resources into healthcare blockchain technology. The Office of [National Coordinator for Health Information Technology (ONC)](https://www.healthit.gov/) is responsible for health information technology. It has recognized a need for nationwide interoperability and standards for electronic health records, claims processing, and verification of provider credentials. To that end, it has sponsored many government blockchain initiatives in healthcare.
```
Comment (tkuhrt) [inline LFS171x/docs/the-promise-of-business-blockchain-technologies.md]: Missed link on `has invested resources`

```
[has invested resources](https://search.usa.gov/search?utf8=%E2%9C%93&affiliate=healthit.gov&query=blockchain&commit=Search)
```

File: LFS171x/docs/the-promise-of-business-blockchain-technologies.md [hunk 14]
```diff
@@ -1 +1,347 @@
...
+
+The healthcare industry has examined placing prescription drug fulfillment processes on blockchain since they involve gathering and checking information from many sources. Insurance benefits investigation eligibility checks are performed to see if insurance will pay. Prior authorization and step therapy requirements are checked to see if a patient is able to receive a particular drug, or if other drugs are preferred. Formulary checks, patients’ assistance checks, and pharmacy stock checks must all be performed. The healthcare industry has also used DLT to handle online identity management, by uploading verified pieces of a healthcare professional’s credentials for license verification.
+
+|![The_BlockRx_Pharma_Ecosystem](../images/the-promise-of-business-blockchain-technologies/The_BlockRx_Pharma_Ecosystem.png)|
+|:--:|
+
+<p align="center">
+  By <a href="https://www.blockrx.com/">The BlockRx Project</a>, retrieved from the <a href="https://wiki.hyperledger.org/groups/healthcare/healthcare-wg">Hyperledger Healthcare Working Group, August 22nd, 2017 Meeting</a>
+</p>
+
```
Comment (tkuhrt) [inline LFS171x/docs/the-promise-of-business-blockchain-technologies.md]: To match other chapters, use:
```
|![The_BlockRx_Pharma_Ecosystem](../images/the-promise-of-business-blockchain-technologies/The_BlockRx_Pharma_Ecosystem.png)|
|:--:|
| *By <a href="https://www.blockrx.com/">The BlockRx Project</a>, retrieved from the <a href="https://wiki.hyperledger.org/groups/healthcare/healthcare-wg">Hyperledger Healthcare Working Group, August 22nd, 2017 Meeting</a>* |
```

File: LFS171x/docs/the-promise-of-business-blockchain-technologies.md (hunk 13 above)
Comment (atc-vince) [reply LFS171x/docs/the-promise-of-business-blockchain-technologies.md]: This particular link was returning a 404 error.

Comment (tkuhrt) [reply LFS171x/docs/the-promise-of-business-blockchain-technologies.md]: Good to know. For an v2 of the course, maybe we should use: https://search.usa.gov/search?utf8=%E2%9C%93&affiliate=usagov&query=healthhit+blockchain

Comment (tkuhrt) [reply LFS171x/docs/the-promise-of-business-blockchain-technologies.md]: Just chatted with Flavia and Magda. They are going to update the course material to use the above link. Can we update the .md file too?

**Comment by atc-vince** [comment]
@tkuhrt Hi Tracy, 
Requested changes implemented.  Links in TOC also tested.
-Vince

**Pull request merged**